### PR TITLE
Flink-BQ connector - Added support for Arrow format

### PR DESCRIPTION
**Pull request by Sridhar-Durgam2022**
Addressed all the review comments . Created PR for the support for Arrow format

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -52,12 +52,19 @@
...
-        <flink.version>1.12.0</flink.version>
+        <flink.version>1.13.1</flink.version>
         <google-cloud-dataproc.version>2.3.1</google-cloud-dataproc.version>
-        <google-cloud-storage.version>2.3.0</google-cloud-storage.version>
+        <google-cloud-storage.version>2.6.1</google-cloud-storage.version>
         <google-truth.version>1.1.3</google-truth.version>
         <guava.version>30.1.1-jre</guava.version>
         <slf4j.version>1.7.15</slf4j.version>
+		<scala.version>2.12</scala.version>
+		<arrow.vector.version>1.0.1</arrow.vector.version>
```
Comment (davidrabinowitz) [inline pom.xml:62]: Should be `arrow.version`. value should be 6.0.1

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]
The PR is too large. Please leave only the Arrow format support files and open additional PRs for other functionalities. 

Use spaces, not tabs in the POM file

Remove redundant dependencies (i.e. lombok)

update dependnecies to latest versions, notice many of them (BigQuery client libraries, arrow, guava, etc) are transitive dependencies from bigquery-connector-common

File: pom.xml [hunk 2]
```diff
@@ -52,12 +52,19 @@
         <avro.version>1.8.2</avro.version>
         <arrow.version>6.0.1</arrow.version>
         <bigquery-connector-common.version>0.23.2</bigquery-connector-common.version>
-        <flink.version>1.12.0</flink.version>
+        <flink.version>1.13.1</flink.version>
```
[the remark this answers is not in the document]
Comment (Sridhar-Durgam2022) [reply pom.xml:55]: We are working on version change. It will be ready in next  1or 2 week

File: pom.xml (hunk 1 above)
Comment (Sridhar-Durgam2022) [reply pom.xml]: changed version

**Comment by Sridhar-Durgam2022** [comment]
Addressed review comments in arrow functionality support

**Pull request closed without merging**